bokun.yaml

```diff
@@ -2707,62 +2708,198 @@ definitions:
     properties:
       questionId:
         type: string
+        description: 'Specifies system ID of the question.'
       questionCode:
         type: string
+        description: 'Specifies external code of the question (created by question owner).'
       label:
         type: string
+        description: 'Specifies question title.'
       help:
         type: string
+        description: 'Explanatory text for this question.'
       placeholder:
         type: string
+        description: 'The placeholder text for this question.'
       dataType:
-        enum:
-        - SHORT_TEXT
-        - LONG_TEXT
-        - INT
-        - DOUBLE
-        - BOOLEAN
-        - DATE
-        - DATE_AND_TIME
-        - OPTIONS
-        type: string
-      dataFormat:
-        enum:
-        - EMAIL_ADDRESS
-        - URL
-        - PHONE_NUMBER
-        - COUNTRY
-        - LANGUAGE
-        - TIME
-        - DAY_OF_MONTH
-        - MONTH
-        - YEAR
-        - PATTERN
-        type: string
-      pattern:
-        type: string
+        $ref: '#/definitions/AnswerDataType'
+        description: 'Data type of the answer object for this question.'
+#      NEED MORE INFO
```

Comment:
You can look up what the actual values for this are by looking at the bokun_platform source code. In this case it is this file that controls this:
https://github.com/Bokun/bokun_platform/blob/master/bokun_api/src/main/java/is/bokun/dtos/questions/QuestionDataTypeEnum.java

I found that by looking at the type definition for the dataType property in https://github.com/Bokun/bokun_platform/blob/master/bokun_api/src/main/java/is/bokun/dtos/questions/QuestionSpecificationDto.java

Reply:
AnswerDataType was documented, it was not the issue.

I was referring to lines 2727-2742, which are commented. Basically, could not find how to make "**dataFormat**" and "**pattern**" properties start working.
I guess the corresponding code is here:
https://github.com/Bokun/bokun_platform/blob/master/app/assets/javascripts/apps/ActivityBookingQuestionForm.react.js

Reply:
```
enum: 
  - SHORT_TEXT
etc...
```

I removed at the same time you commented out the dataFormat property

Reply:
got it. uncommented.